lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1996,6 +1996,23 @@ class DecomposeAtenToDtypeLayoutOp
 };
 } // namespace
 
+namespace {
+// Decompose `aten.to.device` op into `aten.to.dtype` op.
+class DecomposeAtenToDeviceOp : public OpRewritePattern<AtenToDeviceOp> {
```

Comment:
Drive-by comment: I would add a comment here mentioning that once we get to decompositions, the `device` argument no longer means anything in torch-mlir, so we can drop that information.